edx_proctoring/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import
 
-__version__ = '0.18.2'
+__version__ = '0.18.3'
```

Comment:
Okay, I wasn't sure since the last patch version seemed pretty big to me (django upgrade).

1.0.0 sounds good to me :tada: :champagne: :fireworks: :confetti_ball:. If no one else has objections, I'll go bump this to 1.0.0.